packages/config/config/devices/0x024f/fwc01.json

```diff
@@ -0,0 +1,227 @@
+{
+	"manufacturer": "Smartly AS",
+	"manufacturerId": "0x024f",
+	"label": "FWC01",
+	"description": "Wheel Controller",
+	"devices": [
+		{
+			"productType": "0x0003",
+			"productId": "0x1003"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": [
+		{
+			"#": "4",
+			"label": "Safety Activate Delay",
+			"unit": "seconds",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 32767,
+			"defaultValue": 0
+		},
+		{
+			"#": "5",
+			"label": "Safety OFF Period",
+			"unit": "seconds",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 32767,
+			"defaultValue": 600
+		},
+		{
+			"#": "6",
+			"label": "Safety ON Period",
+			"unit": "seconds",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 32767,
+			"defaultValue": 300
+		},
+		{
+			"#": "10",
+			"label": "Dimming speed",
+			"description": "Dimmer Speed in 1/100th of a Second",
+			"unit": "/100s",
+			"valueSize": 2,
+			"minValue": 30,
+			"maxValue": 32767,
+			"defaultValue": 300
+		},
+		{
+			"#": "11",
+			"label": "Dimmer mode",
+			"description": "Dimmer Operation Mode",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1
+		},
+		{
+			"#": "12",
+			"label": "Maximum Level",
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 99,
+			"defaultValue": 90
+		},
+		{
+			"#": "13",
+			"label": "Minimum Level",
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 99,
+			"defaultValue": 15
+		},
+		{
+			"#": "14",
+			"label": "UI modes and Zones",
+			"description": "Enable UI Modes and Zones via Bits",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 67,
+			"defaultValue": 0
+		},
+		{
+			"#": "15",
+			"label": "Light Zone Control Mode",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 30,
+			"defaultValue": 0
+		},
+		{
+			"#": "17",
+			"label": "Back Unit Role",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 2,
+			"defaultValue": 1
+		},
+		{
+			"#": "18",
+			"label": "Back Unit Light Zone",
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 4,
```

Comment:
What do these values mean? There should probably be options describing them.

Reply:
I'll correct it.